Documentation~/boolean.md

```diff
@@ -10,6 +10,7 @@ Each type of operation takes two ProBuilder Mesh objects as the left and right i
 
 You can only access this feature from the menu, because it is experimental:
 
+1. [Enable experimental features](preferences.md#experimental).
 1. To open the Boolean (Experimental) window, navigate to Unity's top menu and go to **Tools** > **ProBuilder** > **Experimental** > **Boolean (CSG) Tool**.
```

Comment:
I think the list has to be re-numbered? There's two "1."s

Reply:
But it would be prettier if they were all `1.`, I suppose.

Reply:
Yeah, there's actually no point in going 1, 2, 3. You can just go 1, 1, 1.

Reply:
Ok, makes sense.